addon/mixins/time-series-labeler.js

```diff
@@ -102,40 +102,33 @@ export default Ember.Mixin.create({
   // The ordering of each time domain from most specific to least specific
   DOMAIN_ORDERING: ['S', 'H', 'D', 'W', 'M', 'Q', 'Y'],
 
-  // This is the number of subdivisions between each major tick on the x
-  // axis. Minor ticks have no labels. For instance, if your maxNumberOfLabels
-  // is 10, and you are charting 20 weeks, there will be 10
-  // major ticks with one subivision (minor ticks) between.
-  numberOfMinorTicks: Ember.computed('xDomain', 'xAxisTimeInterval', 'labelledTicks', function() {
-    var allTicks, domain, findTick, firstIndex, interval, labelledTicks, xDomain, secondIndex, start, stop;
-    labelledTicks = this.get('labelledTicks');
-    xDomain = this.get('xDomain');
-    start = xDomain[0];
-    stop = xDomain[1];
-    domain = this.get('xAxisTimeInterval');
-    interval = domain === 'Q' ? this.MONTHS_IN_QUARTER : 1;
-
-    // All the ticks which occur between start and stop (including
-    // unlabelled ticks)
-    allTicks = d3.time[domainTypeToLabellerType[domain]](start, stop, interval);
-    if (labelledTicks.length < 2) {
-      return 0;
-    }
-
-    // equality for ticks
-    findTick = function(tick) {
-      return function(x) {
-        return +x === +tick;
-      };
-    };
-
-    // Returns the difference between where the second labelled value
-    // occurs in the unlabelled array and where the first occurs - e.g. in
-    // the above example 3 - 1 - 1 => 1 subdivision tick.
-    secondIndex = _.findIndex(allTicks, findTick(labelledTicks[1]));
-    firstIndex = _.findIndex(allTicks, findTick(labelledTicks[0]));
-    return secondIndex - firstIndex - 1;
-  }),
+  // D3 No longer handles "minor ticks" for the user, but has instead reverted
+  // to a strategy of allowing the user to handle rendered ticks as they see
+  // fit.  The new functionality has 2 parts:
+  // 1) Register all of the ticks "filtered" out by filterLabels
+  // 2) Apply a treatment to the labels
+  // maxNumberOfMinorTicks sets a treshold that is useful when determining our
+  // interval.  minorTickInterval is the modulo for the items to be removed.  So
+  // a maxNumberOfMinorTicks=0 and minorTickInterval=1 essentailly disables the
+  // minor tick feature.
+  maxNumberOfMinorTicks: 0,
+  minorTickInterval: 1,
```

Comment:
I'll explain a little better in the comment.  'maxNumberOfMinorTicks' is the number of ticks that _could_ be drawn between major ticks.  The library tries not to render them unless we really need the space.